core/domain/topic_domain.py

```diff
@@ -1212,7 +1297,9 @@ def _convert_subtopic_v1_dict_to_v2_dict(cls, subtopic_dict):
 
     @classmethod
     def update_subtopics_from_model(
-            cls, versioned_subtopics, current_version, topic_id):
+        cls, versioned_subtopics: VersionedSubtopicsDict, current_version: int,
+        topic_id: str
+    ) -> None:
```

Comment:
```suggestion
        cls, 
        versioned_subtopics: VersionedSubtopicsDict, 
        current_version: int,
        topic_id: str
    ) -> None:
```
if the function definition cannot fit three lines just put every argument on a separate line.

Reply:
done.